src/widget/form/settings/generalsettings.ui

```diff
@@ -377,7 +377,7 @@ instead of system taskbar.</string>
           <item>
            <widget class="QCheckBox" name="cbSeparateWindow">
             <property name="text">
-             <string>Open chats in separate window.</string>
+             <string>Use multiple windows</string>
```

Comment:
This wording indicates that with this option unchecked it's not possible to open a detached window, which is not the case, since user still can do that manually by r-clicking.